Automattic/woocommerce-payments
# Fix Payment Method Titles After Checkout

Fixes #2434 

#### Changes proposed in this Pull Request


# Rename UPE Payment Methods

> After a chat with @LevinMedia, we decided that we should drop the WooCommerce Payments suffix from the Order Complete page after checkout.

Before: 
![Wrong](https://cdn-std.droplr.net/files/acc_1104206/d9O9mw)
_Incorrectly titled payment method_

After:
![Right](https://cdn-std.droplr.net/files/acc_1104206/mdrW05)
_Correctly titled payment method_




#### Testing instructions


1. Ensure you have pulled the changes in this PR and the latest version of the Payments Server. Also, please enable the UPE gateway using the WCPay Dev Plugin or simply set _wcpay_feature_upe to a value of 1 in your wp_options table.
2. Add a product to your cart and complete a checkout with either a credit card, Giropay, or Sofort (or preferably all three) using the UPE.
3. Confirm that the payment method is titled correctly on the Order Complete page.
4. That's all!


-------------------
